search.c

```diff
@@ -829,7 +829,16 @@ static void string_tolower(char *str)
   }
 }
 
-static int do_search(struct read_db *db, char **args, char *output_path, int show_threads, enum folder_type ft, int verbose, const char *imap_pipe, const char *imap_server, const char *imap_username, const char *imap_password, int please_clear)/*{{{*/
+static struct read_db *qsortdb;
```

Comment:
A bit smelly to use a global here, but I'm not going to be fussy with this code.

Reply:
Yeah I'd've preferred to use qsort_r, but wasn't sure how you'd react to my locking in (I think) a dependency on GNU libc.  The code has a lot of non-reentrant/global stuff going on.  Might look at bits of it again if I get time.

Thanks for merging!